docs/docs/target/container_image.md

```diff
@@ -170,11 +170,6 @@ See [here](../scanner/secret.md) for the detail.
 ```shell
 $ trivy image --image-config-scanners secret [YOUR_IMAGE_NAME]
 ```
-
-If you just want to scan the image config.For example:
-
-```shell
-$ trivy image --image-config-scanners secret vuln-image
 ```
```

Comment:
This line breaks the document. You can build the doc with `mage docs:serve` and see how it looks.
https://aquasecurity.github.io/trivy/v0.45/community/contribute/pr/#documentation